.github/workflows/ci.yml

```diff
@@ -5,12 +5,6 @@
 name: CI
 on:
```

Comment:
`on: [push]` may be more compact, but semantically this is already what we want.

Reply:
Done